unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs

```diff
@@ -550,6 +539,7 @@ public void ExitEditMode()
         RenderSettings.skybox = previousSkyBoxMaterial;
 
         OnExitEditMode?.Invoke();
+        DataStore.i.appMode.Set(AppMode.DEFAULT);
```

Comment:
Maybe we should take into account for the future that we should be able to go back to the previous mode so we can do a "Pile"  of states. 

For example Defautl state -> BuilderInWorld -> Preview Mode

Reply:
Hmmm maybe in the future we could need something like this, but for now I don't see problem to give the responsibility of set each specific mode to each module (BiW, preview mode, etc.). At the moment I wouldn't like to overimplement the system, if we need it in the future it will not be something difficult to adapt.